kernel/univ.ml

```diff
@@ -320,8 +322,9 @@ struct
       if u == v then 0
       else
         let (x, n) = u and (x', n') = v in
-          if Int.equal n n' then Level.compare x x'
-          else n - n'
```

Comment:
Wait a sec, wasn't that a potential source of unsoundness?

Reply:
Depends hox callers use it I guess.

Reply:
Let's pretend nobody saw anything.